dev/com.ibm.ws.ui.tool.explore/resources/WEB-CONTENT/jsExplore/widgets/templates/BreakAffinity.html

```diff
@@ -0,0 +1,14 @@
+<!--
```

Comment:
In other html files, the copyright comment is not on the first line.  It is here.  Is that a problem?